rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -801,6 +803,46 @@ def assert_okay(response):
             api_url_for(server, 'assetmovementsresource') + '?' + urlencode(data))
         assert_okay(response)
 
+    # now test `exclude_ignored_assets` query param
+    with setup.polo_patch:
+        data = {'only_cache': True}
+        without_ignore_response = requests.get(
+            api_url_for(server, 'assetmovementsresource'),
+            json=data,
+        )
+        without_ignore_response = assert_proper_response_with_result(without_ignore_response)
+
+        toggle_ignore_an_asset(rotkehlchen_api_server_with_exchanges, A_BTC)
+        data |= {'exclude_ignored_assets': True}
+        ignored_response = requests.get(api_url_for(server, 'assetmovementsresource'), json=data)
+        ignored_response = assert_proper_response_with_result(ignored_response)
+        default_response = requests.get(api_url_for(server, 'assetmovementsresource'))
+        default_response = assert_proper_response_with_result(default_response)
+        msg = '`exclude_ignored_assets` should be True by default'
+        assert ignored_response == default_response, msg
```

Comment:
nitipck. Keep things compact here and in other places in the code

```suggestion
        assert ignored_response == default_response, '`exclude_ignored_assets` should be True by default'  # noqa: E501
```

Reply:
Oh I was trying to avoid `# noqa: E501`, but okay.

Reply:
yeah I get it. Some times for compactness it's okay to have it.